docs/how-to/code/create-slice/task.yaml

```diff
@@ -12,11 +12,12 @@ execute: |
   apt show openssl
   # [docs:apt-show-openssl-end]
 
-  git clone -b fix-copyright-symlink https://github.com/woky/chisel
+  git clone https://github.com/canonical/chisel
   git clone -b ubuntu-22.04 https://github.com/canonical/chisel-releases
   cp openssl.yaml chisel-releases/slices
```

Comment:
I guess this `cp` is no longer necessary?

Reply:
oh of course; I had it in my head that this was part of the copyright symlink fix somehow
thanks